pkg/commands/compute/setup/kv_store.go

```diff
@@ -149,11 +181,16 @@ func (o *KVStores) Create() error {
 				msg := fmt.Sprintf("Creating kv store key '%s'...", item.Key)
 				o.Spinner.Message(msg)
 
-				err = o.APIClient.InsertKVStoreKey(&fastly.InsertKVStoreKeyInput{
-					ID:    store.ID,
-					Key:   item.Key,
-					Value: item.Value,
-				})
+				input := &fastly.InsertKVStoreKeyInput{
+					ID:  store.ID,
+					Key: item.Key,
+				}
+				if item.Body != nil {
+					input.Body = item.Body
+				} else {
+					input.Value = item.Value
```

Comment:
I didn't look too closely, but is it possible to get here without Value having been set?

Reply:
So it's hard to see in the diff, but we are actually in a different method here (this is `Create()`). In the earlier `Configure()` method if the user doesn't provide a value for 'value' field, then we set a default of `"example"`.